rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-sts-creating-a-cluster-ext-auth"]
+= Creating ROSA with HCP clusters with external authentication
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-sts-creating-a-cluster-ext-auth
+
+toc::[]
+
+You can create {hcp-title-first} clusters that use an external authentication to issue your access tokens.
+
+[IMPORTANT]
+====
+Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality.
+====
+
+[NOTE]
+====
+{hcp-title} clusters only support AWS Security Token Service (STS) authentication.
+====
+
+.Further reading
+* For a comparison between {hcp-title} and ROSA Classic, see the xref:../architecture/rosa-architecture-models.adoc#rosa-hcp-classic-comparison_rosa-architecture-models[Comparing architecture models] documentation.
+* See the AWS documentation for information about link:https://docs.aws.amazon.com/rosa/latest/userguide/getting-started-hcp.html[Getting started with ROSA with HCP using the ROSA CLI in auto mode].
```

Comment:
This feature is also available in OCP.

I wonder if we could link with their documentation as well.

Perhaps they also have some description of the differences between internal OIDC and external OIDC.

Reply:
Would it be this one for external OIDC? https://docs.openshift.com/container-platform/4.15/authentication/configuring-oauth-clients.html

I'm not extremely familiar with their documentation, but I can dig around if it's not this one.

Reply:
BYO External OAuth is a new feature and not yet available in Standalone OCP.  OIDC IDP integrations are available but they have to integrate with OpenShift OAuth server. https://docs.openshift.com/container-platform/4.15/authentication/understanding-identity-provider.html

Reply:
ah, @anjaltelang is right, if this feature is not available in other OCP sections, my comment doesn't make sense.